### DISCUSSION TITLE
Image component not loading different sized images

### DISCUSSION THREAD

**Discussion by ragnsan** [vercel/next.js#25033] [Help]
I am having an issue where the image component does not load different sized images depending on screen size. It serves the w_3840 version no matter what the screen size is.

The website is statically hosted and the images are hosted on cloudinary. In the image below, you can my next.config.js

![Code_rtoqzAsprL](https://user-images.githubusercontent.com/70635181/117948468-7396a300-b311-11eb-9d76-9a285613aadb.png)

When I use a locally hosted image in the public folder, the w_ in the url request path gets set to one of the deviceSizes in the config. I.e. w_750. But when I use an image from my Cloudinary account, something ads a second set of parametres (c_limit,w_3840 etc.), thus resulting in serving only the w_3480 version.

![chrome_06Qkjj10A4](https://user-images.githubusercontent.com/70635181/117948749-b9ec0200-b311-11eb-9733-72ad84452596.png)

If If you have encountered and solved this issue, please help me out!

Best regards!

**Comment by ragnsan** [comment]
Please help

**Comment by mayankzalavadia** [comment]
We also face the same issue. Can anyone help us to deal with this issue?
